fava/core/budgets.py

```diff
@@ -61,6 +61,19 @@ def parse_budgets(custom_entries):
                                 entry.values[1].value,
                                 entry.values[2].value.number,
                                 entry.values[2].value.currency)
+                if budget.period not in [
+                    "daily", "weekly", "monthly", "quarterly", "yearly"
+                ]:
+                    errors.append(
+                        BudgetError(
+                            entry.meta,
+                            'invalid budget period "%s", please use "daily", \
```

Comment:
Can you make the message translatable?

Reply:
@aumayr: none of the other error messages are translateable - I wouldn't change that here

Reply:
I think we should start to translate the error messages, at least the ones from Fava.